portfolio/src/main/webapp/script.js

```diff
@@ -23,7 +39,7 @@ function getResult() {
     document.getElementById('spotifyPlayer').src = randomSong;
     console.log("recommending");
     //callApi( "GET", _baseUri + "/recommendations?limit=1&market=US&seed_artists=7Ln80lUS6He07XvHI8qqHH&seed_genres=alternative&seed_tracks=5ruzrDWcT0vuJIOMW7gMnW", null, handleGetResultResponse);
-    callApi( "GET", _baseUri + "/recommendations?limit="+songNum+"&market=US&seed_artists="+seedArtists+"&seed_genres="+seedGenres+"&seed_tracks="+seedTracks, null, handleGetResultResponse);
+    callApi( "GET", _baseUri + "/recommendations?limit="+songNum+"&market=US&seed_genres="+genre+"&target_acousticness="+acousticness+"&target_popularity="+popularity+"&target_tempo="+bpm+"&target_valence="+valence, null, handleGetResultResponse);
```

Comment:
I'm guessing this doesn't quite work yet, but a good start so can go ahead and submit.
I think what we actually can do is extract the form values from the url params after the redirect to results.html.
If you submit this though I can merge it into mine and submit that and then we can worry about hooking up the params.